src/rendering/renderers/shared/texture/Texture.ts

```diff
@@ -85,10 +85,10 @@ export type TextureSourceLike = TextureSource | TextureResourceOrOptions | strin
  *
  * ```js
  *
- * const texture = await Asset.load('assets/image.png');
+ * const texture = await Assets.load('assets/image.png');
  *
  * // once Assets has loaded the image it will be available via the from method
- * const sameTexture = Texture.from('assets/image.png');
+ * const sameTexture =await Assets.load('assets/image.png');
```

Comment:
This one should still be Texture.from.
```suggestion
 * const sameTexture = Texture.from('assets/image.png');
```